content/pages/contact.md

```diff
@@ -6,11 +6,11 @@ Online, we primarily communicate through an email [mailing list]({filename}/page
 
 ## Internet Relay Chat
 
-The LUG has an IRC channel which everyone (especially members) is welcome to join. If you are new to IRC, you can connect through freenode's [Web IRC](http://webchat.freenode.net/?channels=nairobilug).
+The LUG has an IRC channel which everyone (especially members) is welcome to join. If you are new to IRC, you can connect through freenode's [Web IRC](https://web.libera.chat/?channels=nairobilug).
```

Comment:
the new libera.chat link doesn't work. It seems like it should be: `https://web.libera.chat/?channels=#nairobilug` to direct to the nairobilug channel.

Reply:
Thanks ...